src/System.Windows.Forms/src/System/Windows/Forms/ListViewItem.ListViewItemAccessibleObject.cs

```diff
@@ -142,26 +145,195 @@ public override void DoDefaultAction()
 
                         break;
                     case UiaCore.NavigateDirection.LastChild:
-                        int subItemsCount = _owningItem.SubItems.Count;
-                        if (subItemsCount > 0)
-                        {
-                            return GetChild(subItemsCount - 1);
-                        }
-
-                        break;
+                        return GetChildInternal(LastChildIndex);
                 }
 
                 return base.FragmentNavigate(direction);
             }
 
             public override AccessibleObject? GetChild(int index)
+                // Only additional ListViewSubItem are displayed in the accessibility tree if the ListView
+                // in the "Tile" view (the first ListViewSubItem is responsible for the ListViewItem)
+                => GetChildInternal(_owningListView.View == View.Tile ? index + 1 : index);
+
+            internal AccessibleObject? GetChildInternal(int index)
             {
-                if (index < 0 || index >= _owningItem.SubItems.Count)
+                // If the ListView does not support ListViewSubItems, the index is greater than the number of columns
+                // or the index is negative, then we return null
+                if (!_owningListView.SupportsListViewSubItems || _owningListView.Columns.Count <= index || index < 0)
                 {
                     return null;
                 }
 
-                return _owningItem.SubItems[index].AccessibilityObject;
+                if (_owningListView.View == View.Details)
+                {
+                    return GetDetailsSubItemOrFake(index);
+                }
+
+                // The ListViewSubItem for "Tile" mode is only displayed if the following conditions are met:
+                // 1. The ListViewSubItem is not the first (data about the first ListViewSubItem is displayed in the ListViewtem itself)
+                // 2. The ListViewSubItem has non-empty bounds(otherwise it means that it is not displayed)
+                if (index > 0
+                    && _owningItem.SubItems.Count > index
+                    && !GetSubItemBounds(index).IsEmpty)
+                {
+                    return _owningItem.SubItems[index].AccessibilityObject;
+                }
+
+                return null;
+            }
+
+            public override int GetChildCount()
+            {
+                if (!_owningListView.IsHandleCreated|| !_owningListView.SupportsListViewSubItems)
+                {
+                    return -1;
+                }
+
+                return _owningListView.View switch
+                   {
+                       View.Details => _owningListView.Columns.Count,
+                       View.LargeIcon => -1,
+                       View.List => -1,
+                       View.SmallIcon => -1,
+                       View.Tile => GetChildCountTileView(),
+                       _ => throw new NotImplementedException()
```

Comment:
This is much safer. In a very unlikely scenario where Windows introduces a new mode we don't want to break consumers.
```suggestion
                       View.Details => _owningListView.Columns.Count,
                       View.Tile => GetChildCountTileView(),
                       _ => -1
```